helpers/constants.py

```diff
@@ -52,6 +52,9 @@
 module_not_found = "Module not found"
 version_url_fragment_warning = "Allowed input for ulrFragment: alphanumeric, hyphen, period and underscore"
 new_product_url_fragment = "Test UrlFragment"
+random = utilities.generate_random_string(4)
```

Comment:
@noopurmath3 this is not the right place to use a function, this file contains only the constants, we cannot have a random string generator in the middle of constants. please use this function somewhere in other modules or maybe a separate function to do the same, refer api-tests suite for consistency.

Reply:
Marking this resolved as per conversations on chat